infrastructure/cli/pkg/cmd/create/create.go

```diff
@@ -48,14 +49,33 @@ func create(cmd *cobra.Command, args []string) {
 		fmt.Println("setting up Helm failed with err: ", err)
 		os.Exit(1)
 	}
-	if err := helm.InstallCharts(); err != nil {
+	if err := helm.InstallCharts(provider.GetHelmOverrides()); err != nil {
 		fmt.Println("installing Helm charts failed with err: ", err)
 		os.Exit(1)
 	}
 
+	viper.Set("KubeConfig", context.KubeConfigPath)
+	viper.Set("ContextName", context.ContextName)
+	if err = viper.WriteConfig(); err != nil {
+		fmt.Println("could not store the kube context: ", err)
+		os.Exit(1)
+	}
+
 	fmt.Println("🚀 Starting core with default components")
 	fmt.Println("🎉 Your Airy Core is ready")
-	fmt.Println("\t Link to the API")
-	fmt.Println("\t Link to the UI")
-	fmt.Println("\t Link to more docs")
+
+	hosts, err := provider.GetHosts()
+	if err != nil {
+		fmt.Println("failed to get installation endpoints: ", err)
+		os.Exit(1)
+	}
+
+	fmt.Println("\t Available hosts:")
+	for hostName, host := range hosts {
+		fmt.Printf("\t\t %s:\t %s", hostName, host)
+	}
+
+	fmt.Println()
+	fmt.Printf("For more information about the %s provider visit https://airy.co/docs/core/getting-started/installation/%s", providerName, providerName)
```

Comment:
same question. Very nice touch with the docs :)

Reply:
Good question. Will put 📚  for now

Reply:
Wondering if we should start doing colored terminal mockups as part of our design process 🤔

Reply:
🤓 or 📚 ?

Reply:
No strong opinion 😄 What does the team think?